stack/corepersistence/collection/src/main/java/org/apache/usergrid/persistence/collection/serialization/impl/MvccEntitySerializationStrategyImpl.java

```
@@ -318,26 +327,33 @@ public ByteBuffer toByteBuffer( final EntityWrapper wrapper ) {
                 builder.addBytes( STATE_PARTIAL );
             }
 
-            builder.addBytes( SER.toByteBuffer( wrapper.entity.get() ));
+            try {
+                builder.addBytes( mapper.writeValueAsBytes( wrapper.entity.get() ) );
+            }
+            catch ( JsonMappingException e ) {
```

Don't swallow exceptions.  If we get something weird here, wrap it in a CollectionRuntimeException and re-throw it.  We always want to signal to the caller something failed if we had a problem.